### DISCUSSION TITLE
Ridgeclassifier error Pycaret classifier

### DISCUSSION THREAD

**Discussion by binary-alkemi** [pycaret/pycaret#3353] [Q&A]
Hi,

I am trying the Pycaret classifier as:

```
exp_class101 = setup(data=df1 , target = 'pubchem_activity', session_id=123,
                     use_gpu=True, fold_shuffle=True, preprocess=False) 

```
However, I ended up getting error:

`AssertionError: Error in `get_ridge_classifier.<locals>._RidgeClassifierBase.__init__`!. Positional arguments for estimators (that derive from `Base`) have been deprecated but parameters 'alpha' can still be used as positional arguments. Please specify all parameters after `self` as keyword only by using the `*` argument`

I assume this error is arising due to a deprecated parameter.

Can we bypass this error or avoid specific classifiers in Pycaret?

Thanks in Advance,
Mandar Kulkarni

**Comment by Yard1** [comment]
Hi, this should have been fixed in https://github.com/pycaret/pycaret/pull/3286. Can you update pycaret with `pip install -U --pre pycaret` and try again?
